DOCS/new-license-workflow.md

```diff
@@ -17,23 +24,154 @@ Ask the submitter for any additional info needed, preferably via the Github issu
-    3. Once the XML is done, “submit changes” in the tool, which will create a new PR in the repo. Tag the PR to be reviewed by at least one other member of the legal team before merging.
-    4. Once the PR is created in the repo, add a .txt file for the license in the test/simpleTestForGenerator directory
-    4. Check to see if the PR passes the automated test suite on check-in.  If it does not pass, evaluate the cause and resolve it.  If you need assistance, contact one of the technical leads (@goneall or @zvr) for assistance.
-    4. Once review has been completed and there are no further question, merge PR, and close issue.
+
+If the license/exception is accepted, make a note in the issue, add the [label](https://github.com/spdx/license-list-XML/labels) `new license/exception: Accepted`. 
+
+The final step is to create the license XML and test .txt files.
+
+While you can do this using the [SPDX Online tools](https://spdxtools.sourceauditor.com/), we recommend you [clone (fork) the license-list-XML repository (repo)](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-forks), make the edits on your clone of the repo, then [send a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork). This will be necessary for the test .txt file regardless, as the SPDX Online tools do not yet support adding a test .txt file.
+
+The following steps assume you're working from a clone of the repo, but we have some basic steps afterward if you'd like to edit the XML using the SPDX Online tools.
+
+> ⚠️ **NOTA BENE:** All files rely on the `licenseId` (short identifier) for the license. The XML and test .txt files must be named identically using that `licenseId` value. For instance, if you're adding the _K-9 Robotic Dog Hardware License_ with a `licenseId` of _K-9RDHL_, you will have an XML file named `K-9RDHL.xml` and a test .txt file named `K-9RDHL.txt`.
+
+### Add the test .txt file
+
+1. The test files are in the `test/simpleTestForGenerator/` directory of your clone of the license-list-XML repo.
+1. Locate the canonical text for the license. There should be a link to this in the issue, but if there isn't please ask for it from the license steward. Don't proceed until you have confirmed that you have the canonical text.
+1. Copy the text of the license and add it to an appropriately named .txt file (see above for naming guidelines). This must be plaintext and ASCII/Unicode with no special characters such as smart quotes. Do try to keep formatting elements such as section indentation, _using spaces to make the indentation rather than using tabs_. For example, here's one of many indented sections in the [CC-BY-NC-ND-3.0-IGO.txt file](../test/simpleTestForGenerator/CC-BY-NC-ND-3.0-IGO.txt):
```

Comment:
The tool that compares and publishes the license list data assumes the .txt file is encoded as UTF-8.  It handles many of the special characters, but there are probably some out there still that may trip up the compare, so recommending the avoidance of special characters is a good idea.

Suggest changing "This must be plaintext and ASCII/Unicode with no special characters such as smart quotes" to "This must be UTF-8 encoded.  Special characters such as smart quotes should be avoided.

Reply:
I think the sentence @goneall proposes makes sense.